packages/sdk/src/types/attributes/IQLQuery.ts

```diff
@@ -1,5 +1,5 @@
 export interface IQLQuery {
-    "@type"?: "IQLQuery";
+    "@type": "IQLQuery";
```

Comment:
This is a breaking change where I don't see the reason. Could this be a bug within the Runtime?